webapp/src/pages/MapInfo.tsx

```diff
@@ -0,0 +1,126 @@
+import PODManager from "../adapters/solid/PODManager";
+import SolidSessionManager from "../adapters/solid/SolidSessionManager";
+import UserStuff from "./UserStuff";
+import PrivacyComponent from "../components/place/PrivacyComponent";
+import User from "../domain/User";
+import Placemark from "../domain/Placemark";
+import PointInformation from "./PointInformation";
+import LoadingPage from "../components/basic/LoadingPage";
+
+interface MapInfoProps {
+    map: Map;
+}
+
+interface MapInfoState {
+    goBack: boolean;
+    visibility: string;
+    pod: PODManager;
+    theMap: Map;
+    open: boolean;
+    selectedPlacemark: Placemark;
+    currentComponent: JSX.Element;
+    loadedPlaces: boolean;
+}
+
+export default class MapInfo extends React.Component<MapInfoProps, MapInfoState> {
+
+    private sessionManager: SolidSessionManager = SolidSessionManager.getManager();
+
+    public constructor(props: any) {
+        super(props);
+        this.state = {
+            goBack: false,
+            visibility: "",
+            pod: new PODManager(),
+            theMap: this.props.map,
+            open: false,
+            selectedPlacemark: new Placemark(0, 0),
+            currentComponent: <div/>,
+            loadedPlaces: false
+        };
+        this.goBack = this.goBack.bind(this);
+        this.handleVisibilityChange = this.handleVisibilityChange.bind(this);
+        this.handleBack = this.handleBack.bind(this);
+    }
+
+    private async loadPlaceMarks(callback: () => void) {
+        await this.state.pod.loadPlacemarks(this.state.theMap);
+        this.state.theMap.getPlacemarks().forEach((placemark) => {
+            console.log(placemark.getTitle());
+        });
+        callback();
+    }
+
+    private goBack() {
+        this.setState({goBack: true});
+    }
+
+    private handleVisibilityChange = (privacy: string, friends: User[]) => {
+        
+    }
+
+    public async componentDidMount() {
+        await this.loadPlaceMarks(() => {
+            this.setState({theMap: this.state.theMap, loadedPlaces: true});
+        });
+    }
+
+    handleBack(prevComponent: JSX.Element) {
+        this.setState({open: false});
+    }
+
+    /**
+     * Returns the map information view
+     */
+    public render() {
+        if (this.state.goBack) {
+            return <UserStuff/>;
+        }
+        return (
+            <section className="my-stuff">
+                <div className="mapInformation">
+                    <h1>Title: {this.state.theMap.getName()}</h1>
+                    <h2>Description: {this.state.theMap.getDescription()}</h2>
+                    <h2>Places:</h2>
+                    {/*For each place from the map an item will be a row with the name in a table, and a button to open a component PointInformation*/}
+                    {this.state.loadedPlaces &&
+                        <div className="places-buttons" >
+                            {this.state.theMap.getPlacemarks().map((placemark) => (
```

Comment:
If it is possible, write a point before each name with the purpose of ahowing them like a list